tidb-cloud/serverless-limitations.md

```diff
@@ -22,6 +22,7 @@ We are constantly filling in the feature gaps between {{{ .starter }}}/Essential
 
 - Only [Public Endpoint](/tidb-cloud/connect-via-standard-connection-serverless.md) and [Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md) can be used. You cannot use [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md) to connect to {{{ .starter }}} or {{{ .essential }}} clusters.
 - No [Firewall Rules](/tidb-cloud/configure-serverless-firewall-rules-for-public-endpoints.md) support for Private Endpoint.
+- Your database client connections might be terminated unexpectedly if they remain open for more than 30 minutes. This can happen when a server shuts down or restarts, leading to application disruptions. To prevent this, you should configure a maximum connection lifetime. We recommend starting with 5 minutes and increasing it if there is an impact on tail latency. For more details, see [Recommended settings for connection pools](/develop/dev-guide-connection-parameters.md#connection-pools-and-connection-parameters).
```

Comment:
```suggestion
- Your database client connections might be terminated unexpectedly if they remain open for more than 30 minutes. This can occur when a TiDB server shuts down, restarts, or undergoes maintenance, potentially causing application disruptions. To avoid this issue, configure a maximum connection lifetime. It is recommended to start with 5 minutes and increase it gradually if it affects tail latency. For more information, see [Recommended settings for connection pools](/develop/dev-guide-connection-parameters.md#connection-pools-and-connection-parameters).
```